command/hcl2_upgrade.go

```diff
@@ -64,10 +64,19 @@ const (
 # once they also need to be in the same folder. 'packer inspect folder/'
 # will describe to you what is in that folder.
 
+# Avoid mixing go templating calls ( for example '{{ upper("string") }}' ) 
```

Comment:
```suggestion
# Avoid mixing go templating calls ( for example ```{{ upper `string` }}```
```

Reply:
Ah cool one thanks I was trying to find a good solution there 🙂 .

Reply:
Ah nah this doesn't work: going to to this:
```suggestion
# Avoid mixing go templating calls ( for example ` + "```{{ upper(`string`) }}```" + ` ) 
```